junit/MatrixTest.java

```diff
@@ -260,7 +261,38 @@ public void testCtorFillWithConstb()
 
     //Matrix copy(){return null;}
 
-    //double get(int i, int j){return 0;}
+    /**
+     * testGet -Tests the functionality of the get method
+     *  in retrieving the element at the specified index.
+     */
+    @Test
+    public void testGet(){
+        double[][] testMatrix = {{1,2,3,4},{5,6,7,8},{9,10,11,12}};
+        Matrix matrix = new Matrix(testMatrix);
+        int[] testIndex = new int[2];
+        Random rand = new Random();
+        //create the index 'i' for which to test equivalency
+        testIndex[0] = rand.nextInt(2) + 0;
```

Comment:
I thought that the '+ x' sets the minimum value so that: `rand.nextInt(20) + 5` produces integers of range 5 - 20 inclusive. This may be incorrect, i'll look it up early tomorrow morning.